README.md

```diff
@@ -1,22 +1,174 @@
+![a-slice-of-ubuntu](https://canonical-rockcraft.readthedocs-hosted.com/en/latest/_images/slice-of-ubuntu.png)
+
+This image depicts a simple case, where both packages A and B are deconstructed
+into multiple slices. At a package level, B depends on A, but in reality, there
+might be files in A that B doesn’t actually need (eg. A_slice3 isn’t needed for
+B to function properly). With this slice definition in place, Chisel is able to
+extract a highly-customized and specialized Slice of the Ubuntu distribution,
+which one could see as a block of stone from which we can carve and extract
+small and relevant parts we need to run our applications. It is ideal to
+support the creation of smaller but equally functional container images.
+
+> “The sculpture is already complete within the marble block, before I start my
+> work. It is already there, I just have to chisel away the superfluous
+> material.”
+>
+> \- Michelangelo
+
+In the end, it’s like having a slice of Ubuntu - get *just what you need*. You
+*can have your cake and eat it too*!
+
+## Using Chisel
+
+To install the latest version of Chisel, run the following command:
+
+```bash
+go install github.com/canonical/chisel/cmd/chisel@latest
 ```
-$ chisel cut --release release/ --root output/ mypkg_bins mypkg_config
+
+Chisel is invoked using `chisel <command>`. To get more information:
+
+ - To see a help summary, type `chisel -h`.
+ - To see a short description of all commands, type `chisel help --all`.
+ - To see details for one command, type `chisel help <command>` or
+`chisel <command> -h`.
+
+### Example command
+
+Chisel relies on a [database of slices](https://github.com/canonical/chisel-releases) that are indexed per Ubuntu release.
+
+```bash
+chisel cut --release ubuntu-22.04 --root myrootfs/ libgcc-s1_libs libssl3_libs
 ```
 
-## Example release configuration
+In this example, Chisel would look into the Ubuntu Jammy archives, fetch the
+provided packages and install only the desired slices into the *myrootfs*
+folder, according to the slice definitions available in the
+["ubuntu-22.04" chisel-releases branch](<https://github.com/canonical/chisel-releases/tree/ubuntu-22.04>).
+
+## Reference
+
+### Chisel commands
+
+#### help
+
+Show basic help summary. Can also be used to display command details.
+
+#### version
+
+Displays the Chisel version.
+
+#### cut
+
+Uses the provided selection of package slices, for the given release, to create
+a new filesystem tree in the root location.
+
+### Chisel releases
+
+As mentioned above, Chisel relies on **Package Slices**. These slices need to
+be defined prior to the execution of the `chisel` command.
+
+By default, Chisel will look into its central [chisel-releases](https://github.com/canonical/chisel-releases)
+database, where package slices are defined and indexed by Ubuntu release.
+
+One can, however, also point Chisel to a custom and local Chisel release (i.e.
+`chisel cut --release ubuntu-22.10 ...` will fetch the package slices from
```

Comment:
Connecting this paragraph to the earlier one, but then moving the commands into their own code blocks, might make the flow slightly better. Worth testing.